DeeperFunc/PartialIso.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  PartialIso.swift
+//  DeeperFunc
+//
+//  Created by Ilya Puchka on 31/10/2017.
+//  Copyright © 2017 Ilya Puchka. All rights reserved.
+//
+
+import Foundation
+
+infix operator >>>
+infix operator <<<
+
+public struct PartialIso<A, B> {
```

Comment:
Does this really improve readability comparing with just passing apply/unapply function pairs and making some type casting in them?

Reply:
Decided to throw this type away as just using pair of closures is easier to read and I don't need it as a standalone type